spec/unit/crypto/algorithms/olm.spec.ts

```diff
@@ -1,6 +1,6 @@
 /*
```

Comment:
There appear to be some other untyped `let`s in this file too (I can't highlight them because they're outside the diff)